sources/Core/Numerics/Matrix3x3.cs

```diff
@@ -111,5 +151,16 @@ public string ToString(string? format, IFormatProvider? formatProvider)
                 .Append('>')
                 .ToString();
         }
+
+        /// <summary>Creates a transposed version of this <see cref="Matrix3x3" />.</summary>
+        /// <returns>The transposed <see cref="Matrix3x3" />.</returns>
+        public static Matrix3x3 Transpose(Matrix3x3 m)
```

Comment:
```suggestion
        public static Matrix3x3 Transpose(Matrix3x3 value)
```

Rest of this file looks good.

Reply:
fixed